espn_api/base_league.py

```diff
@@ -92,6 +97,17 @@ def _get_all_pro_schedule(self):
             pro_team_schedule[team['id']] = pro_game
         return pro_team_schedule
 
+    def _get_team_owners(self, data) -> dict:  
+        team_owners = {}
+        for team in data['teams']:
+            team_owners[team['id']] = []
+            for member in self.members:
+                for owner in team['owners']:

```

Comment:
team might not have an owner, I would suggest using `.get` to get `owners` filed and falling back to empty array